less/footer.less

```diff
@@ -1,12 +1,23 @@
 .footer {
-  width: 100%;
+  width: 100vw;
+  max-width: inherit;
+  margin-top: -30px;
+  position: absolute;
```

Comment:
There is currently an issue with `100vw` wherein all browsers except Firefox incorrectly cause a horizontal scrollbar to appear because the vertical scrollbar is considered to be part of the viewport. For this reason, I would recommend sticking with your previous idea of using `width: 100%`.

More info about the `100vw` issue here: https://caniuse.com/#feat=viewport-units